opencti-platform/opencti-graphql/src/database/stix.ts

```diff
@@ -867,95 +860,6 @@ export const stixCoreRelationshipsMapping: RelationshipMappings = {
     { name: RELATION_USES, type: REL_EXTENDED }
   ],
   // endregion
-  // region THREAT_ACTOR_INDIVIDUAL
```

Comment:
TAI defs are inside the dedicated module, these defs are all overwritten (I double checked)